spaceship/lib/spaceship/portal/device.rb

```diff
@@ -148,6 +148,8 @@ def create!(name: nil, udid: nil, mac: false)
             raise "You cannot create a device without a device_id (UDID) and name"
           end
 
+          raise "Device name must be 50 characters or less. \"#{name}\" has a #{name.length}-character length." if name.length > 50
```

Comment:
This is a bit nit picky but can we change `#{name.length}-character` to `#{name.length} character` 🙃